include/lemlib/chassis/differential.hpp

```diff
@@ -247,20 +249,20 @@ class Differential : public Chassis {
          * default to global value
          * @param maxSpeed the maximum speed the robot can move at. 127 at default
          */
-        void moveTo(float x, float y, float theta, int timeout, bool reversed = false, float chasePower = 0,
+        void moveTo(Length x, Length y, Angle theta, Time timeout, bool reversed = true, float chasePower = 0,
```

Comment:
```suggestion
        void moveTo(Length x, Length y, Angle theta, Time timeout, bool reversed = false, float chasePower = 0,
```
Change default parameter value to false